get_ipv6_neighbors_table()

pierky opened the pull request:
See also https://github.com/napalm-automation/napalm-base/pull/311
EDIT: tests should be ok when using the base package from the PR above.

pierky committed:
add get_ipv6_neighbors_table()

pierky committed:
ignore INCMP entries

Example:
2001:DB8:1:2::3:4                 0 -               INCMP Gi0/0/2

ktbyers commented:
@pierky FYI, I am going to wait until the one in base gets approved before reviewing this.

pierky committed:
age is in seconds

but Cisco output in minutes

Time (in minutes) since the address was confirmed to be reachable.

https://www.cisco.com/c/en/us/td/docs/ios-xml/ios/ipv6/command/ipv6-cr-book/ipv6-s4.html#wp1680937550

ktbyers commented:
Will reimplement in reunified napalm.
